assignment.md

```diff
@@ -11,63 +11,76 @@ If the version control concepts or `git` commands are still confusing to you, co
        *.exe                                                                                                                                                     *.o                                                                                                                                                       *.so                                                                                                                                                                                                                                                                                                                # Packages #                                                                                                                                              ############                                                                                                                                                                                                       *.7z                                                                                                                                                      *.dmg                                                                                                                                                     *.gz                                                                                                                                                      *.iso                                                                                                                                                     *.jar                                                                                                                                                     *.rar                                                                                                                                                     *.tar                                                                                                                                                     *.zip                                                                                                                                                                                                                                                                                                               # Logs and databases #                                                                                                                                    ######################                                                                                                                                    *.log                                                                                                                                                     *.sql                                                                                                                                                     *.sqlite 
```

Comment:
you literally added all the things you won't have to care about lol... `.DS_Store`, `.vscode` are way more useful global ignores for python projects